sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -4,7 +4,7 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG d6af758779db6eebdc419fd5e249302f566eb5de)
+  set(UNIFIED_RUNTIME_TAG 7d22f806c45872dd6326a123915fe3c75be25b0e)
```

Take a few more UR changes:
```suggestion
  set(UNIFIED_RUNTIME_TAG 7fa23fc8e6fbe1ac1de00b37d8f74620034bcb04)
```